howso/scikit/scikit.py

```diff
@@ -2,20 +2,17 @@
 from typing import Any, Dict, List, Optional, Union
 import uuid
 
-from howso import engine
-from howso.client import AbstractHowsoClient, HowsoPandasClient
-from howso.client.exceptions import (
-    HowsoApiError,
-    HowsoError,
-    HowsoNotUniqueError,
-)
-import howso.utilities as utils
-from howso.utilities.feature_attributes import infer_feature_attributes
 import numpy as np
 import pandas as pd
 from sklearn.base import BaseEstimator
 from sklearn.metrics import accuracy_score, r2_score
 
+from howso import engine
```

Comment:
These are non-std libraries and should be sorted with the others. If they were relative imports, they'd be in a separate group.

Reply:
`howso` is in the flake8 config as an application_import_name and is grouped with the relative imports as a result.